go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/google/uuid v1.6.0
 	github.com/goreleaser/goreleaser v1.26.2
 	github.com/gorilla/securecookie v1.1.2
+	github.com/hashicorp/go-multierror v1.1.1
```

Comment:
Sorry for the long time it took me to go back at this PR.

The only thing I'm not ok with is the usage of library coming from hashicorp. (regarding the changes they made on their license), and besides introducing a new lib for managing an array of error, I find it a bit overkill.

Reply:
I hesitated creating a helper multierror in the project directly, look at go-multierror codebase, it is light with very few updates/CVEs. I understand each projects have their own preference, if you prefer less dependency, I could replace it.